js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedScheduleRow.tsx

```diff
@@ -81,12 +86,35 @@ export const VirtualizedScheduleRow = (props: ScheduleRowProps) => {
     },
     notifyOnNetworkStatusChange: true,
   });
-  useBlockTraceOnQueryResult(queryResult, 'SingleScheduleQuery');
 
-  useDelayedRowQuery(querySchedule);
-  useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
+  useBlockTraceOnQueryResult(scheduleQueryResult, 'SingleScheduleQuery');
+
+  const [queryScheduleAssetSelection, scheduleAssetSelectionQueryResult] = useLazyQuery<
+    ScheduleAssetSelectionQuery,
+    ScheduleAssetSelectionQueryVariables
+  >(SCHEDULE_ASSET_SELECTIONS_QUERY, {
+    variables: {
+      scheduleSelector: {
+        repositoryName: repoAddress.name,
+        repositoryLocationName: repoAddress.location,
+        scheduleName: name,
+      },
+    },
+  });
+
+  useBlockTraceOnQueryResult(scheduleAssetSelectionQueryResult, 'ScheduleAssetSelectionQuery');
```

Comment:
Would be nice to link to a datadog RUM view of this page before/after to get a sense of the effect on display done.

Reply:
@salazarm and I confirmed that the queries take roughtly the same amount of time, so it doesn't look like there's any added wait time.